src/worker/plugins/run.ts

```diff
@@ -15,9 +17,11 @@ export async function runPlugins(server: PluginsServer, event: PluginEvent): Pro
 
             try {
                 returnedEvent = (await processEvent(returnedEvent)) || null
+                pluginsRan.push(pluginConfig.plugin?.name)
```

Comment:
I'd somehow add an ID here, so we could quickly look up what's needed. The id of a `pluginConfig` is probably the best. Something simple, just like: `${pluginConfig.plugin?.name} (${pluginConfig.id})` is probably enough.